app/models/metric/long_term_averages.rb

```diff
@@ -62,7 +62,8 @@ def self.get_averages_over_time_period(obj, options = {})
       Metric::Aggregation::Process.average(c, nil, results[:avg], counts)
 
       begin
-        results[:dev][c]  = vals[c].stddev
+        results[:dev][c] = vals[c].length == 1 ? 0 : vals[c].stddev
```

Comment:
I'm leaning towards raising an exception vs returning 0 or 0.0 for empty or single item sets.  I don't know how this data is being used, but it feels like 0/0.0 is incorrect.  Even python raises an exception on stdev of empty or single item sets.  Our code consumers of the stdev value in this case should handle the lack of a `stddev`.  Maybe?

```python
>>> import statistics
>>> statistics.stdev([])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/statistics.py", line 575, in stdev
    var = variance(data, xbar)
  File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/statistics.py", line 513, in variance
    raise StatisticsError('variance requires at least two data points')
statistics.StatisticsError: variance requires at least two data points

>>> statistics.stdev([1])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/statistics.py", line 575, in stdev
    var = variance(data, xbar)
  File "/usr/local/Cellar/python3/3.5.1/Frameworks/Python.framework/Versions/3.5/lib/python3.5/statistics.py", line 513, in variance
    raise StatisticsError('variance requires at least two data points')
statistics.StatisticsError: variance requires at least two data points
```